packages/testcontainers/src/containers/RegTestContainer/SPV.ts

```diff
@@ -6,7 +6,7 @@ import { RegTestContainer } from './'
 export class SPV {
   static EXPIRATION = 10
   private readonly client: RegTestContainer
-  private assummedSpvHeight: number
+  private readonly assummedSpvHeight: number
```

Comment:
```suggestion
  private readonly assummedSpvHeight: number
```

Is this safe to remove?